plugins/spu2-x/src/DplIIdecoder.cpp

```diff
@@ -101,10 +101,10 @@ void ProcessDplIISample32(const StereoOut32 &src, Stereo51Out32DplII *s)
     float VL = L / (1 - B); // if B>0, it means R>L, so increase L, else decrease L
     float VR = R / (1 + B); // vice-versa
 
-    // 1.73+1.22 = 2.94; 2.94 = 0.34 = 0.9996; Close enough.
+    // 1.73+1.22 = 2.94; 2.94 * 0.34 = 0.9996; Close enough.
     // The range for VL/VR is approximately 0..1,
     // But in the cases where VL/VR are > 0.5, Rearness is 0 so it should never overflow.
-    const float RearScale = 0.34f * 2;
+    const float RearScale = 0.34f; // *2; Let's remove the times 2 and enjoy the rear audo sans the horrible static noise.
```

Comment:
Is it supposed to be correct with the x2 or not? If the x2 was a coding mistake, then the comment isn't really useful.

Also typo in commit title.